### ISSUE TITLE
Build Fails

### ISSUE THREAD

**Issue by ghost** [zbjornson/node-canvas-webp#2]
The build fails with the following error:

```
/usr/bin/ld: ../libwebp/linux-x86_64/libwebp.a(libwebpencode_la-picture_enc.o): unrecognized relocation (0x2a) in section `.text'
/usr/bin/ld: final link failed: Bad value
```

Apparently the gcc version is outdated. So if anyone could fix that that'd be great.
Thanks.

**Comment by Xetera** [comment]
I have no idea how we managed to find each other on the same hour 3 years after this repo last had any activity but I fixed the build issues in a fork and released the library under `canvas-webp` in https://github.com/Xetera/node-canvas-webp

**Comment by zbjornson** [comment]
Fixed by #3, thanks @Xetera !
